### PR TITLE
Adding timeout variable

### PR DESCRIPTION
**Pull request by mike-walters**
Adding timeout variable to module to resolve issues with GCP (where cloud function defaults to 60s).
The default value for both gcp and aws is set to 300 to match the existing value set in ckr_aws/main.tf - this should hopefully prevent other GCP users experiencing the problem while still offering any flexibility required

cloud function terraform docs: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloudfunctions_function

### CODE REVIEW

**Review by JackTreble — APPROVED** [review]
LGTM

**Review by enicholson125 — COMMENTED** [review]
This looks great, thanks for doing this! Would you mind bumping the minor version in module_version.txt for the GCP module? Otherwise it looks good

File: tf_module/ckr_gcp/main.tf [hunk 1]
```diff
@@ -68,6 +68,7 @@ resource "google_cloudfunctions_function" "key_rotator_cloud_function" {
   trigger_http          = true
   entry_point           = "Request"
   service_account_email = google_service_account.key_rotator_service_account.email
+  timeout = var.ckr_timeout
```
Comment (enicholson125) [inline tf_module/ckr_gcp/main.tf:71]: The linter's going to complain about this one - would you mind running `terraform fmt` over the module?

Comment (mike-walters) [reply tf_module/ckr_gcp/main.tf]: The formatting should be correct now

**Comment by eversC** [comment]
I'll push these changes to a new branch so we can get the PR checks to run

**Comment by eversC** [comment]
fixed/merged in #361

**Pull request closed without merging**